astropy/coordinates/representation.py

```diff
@@ -1660,19 +1667,23 @@ def transform(self, matrix):
 
         return rep
 
-    def __mul__(self, other):
-        self._raise_if_has_differentials('multiplication')
-        return self._dimensional_representation(lon=self.lon, lat=self.lat,
-                                                distance=1. * other)
-
-    def __truediv__(self, other):
-        self._raise_if_has_differentials('division')
-        return self._dimensional_representation(lon=self.lon, lat=self.lat,
-                                                distance=1. / other)
+    def _scale_operation(self, op, *args):
+        return self._dimensional_representation(
+            lon=self.lon, lat=self.lat,
+            distance=1., differentials=self.differentials)._scale_operation(op, *args)
 
     def __neg__(self):
-        self._raise_if_has_differentials('negation')
-        return self.__class__(self.lon + 180. * u.deg, -self.lat, copy=False)
+        if any(differential.base_representation is not self.__class__
+               for differential in self.differentials.values()):
+            return super().__neg__()
+
+        result = self.__class__(self.lon + 180. * u.deg, -self.lat, copy=False)
+        for key, differential in self.differentials.items():
+            new_comps = (op(getattr(differential, comp))
+                         for op, comp in zip((operator.pos, operator.neg),
+                                             differential.components))
+            result.differentials[key] = differential.__class__(*new_comps, copy=False)
```

Comment:
Fine here since we use `diff` as shorthand elsewhere, but I think we do have a 100 char line length in astropy https://github.com/astropy/astropy/blob/main/setup.cfg#L153

Reply:
Good to know. I've been truncating my lines too much!

Reply:
I still like 80 characters... Means I can get 2 windows side-by-side

Reply:
But here I'll let it slide - it just fits in my window :smiley_cat: